There has been a change in the method onResponse, in the section of code starting on line nr 68.
  
The change is in a catch statement, in a try statement, in the method ```onResponse```, in the class ```1```, in an initialization, in the invocation ```exists```, in the method ```handleRequest```, in the class ```RestIndicesExistsAction```.
  
The following changes have been made:  
Changed type reference from ```java.lang.Exception``` to ```Throwable``` on line 68.  
